src/test/scala/com/fasterxml/jackson/module/scala/introspect/ScalaAnnotationIntrospectorTest.scala

```diff
@@ -241,6 +244,12 @@ class ScalaAnnotationIntrospectorTest extends FixtureAnyFlatSpec with Matchers {
     cache.get(new ClassKey(classOf[CaseClassWithDefault])) should not be(null)
   }
 
+  it should "ignore a generated default argument method" in { mapper =>
+    val bean = new GeneratedDefaultArgumentClass
+    val allProps = getProps(mapper, bean)
+    allProps shouldBe empty
+  }
```

Comment:
Is there an argument that `getValue` should be serialized? I understand how `getValue$default$` (or whatever the extra generated method is called) should be ignored but maybe this highlights a 2nd issue.

Reply:
I've merged the change as is (and thanks for submitting it) but I'd still be interested in discussing the point about the `getValue` method.

Reply:
I wouldn't expect `getValue` to be serialised, since it takes a parameter. Even though there is a default value for it, it still seems conceptually different than a property getter. To me, leaving it out seems like the least surprising choice. I don't feel strongly about this, though… I could be convinced otherwise.

Reply:
good point - thanks